pages/installation/os/centos.rst

```diff
@@ -85,6 +85,9 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl start graylog-server.service
 
 
+.. note:: We higly recommend that you proxy the Graylog Webinterface and API with NGINX/Apache like described in: :ref:`configuring_webif_nginx`.
```

Comment:
I think we should remove this note and rather improve our own HTTPS documentation.

Otherwise we could as well remove HTTPS support from Graylog and always recommend using a 3rd party reverse proxy.